Formula/pyqt.rb

```diff
@@ -13,51 +12,56 @@ class Pyqt < Formula
     sha256 cellar: :any, mojave:        "0ad16dcf14f820bb1ecbd96974264d04f9c58ee8f8068cd7f47de88b275d8c1e"
   end
 
+  depends_on "pyqt-builder" => :build
   depends_on "python@3.9"
-  depends_on "qt@5"
+  depends_on "qt"
   depends_on "sip"
 
-  resource "PyQt5-sip" do
-    url "https://files.pythonhosted.org/packages/73/8c/c662b7ebc4b2407d8679da68e11c2a2eb275f5f2242a92610f6e5024c1f2/PyQt5_sip-12.8.1.tar.gz"
-    sha256 "30e944db9abee9cc757aea16906d4198129558533eb7fadbe48c5da2bd18e0bd"
+  resource "PyQt6-sip" do
+    url "https://files.pythonhosted.org/packages/95/7e/495d128a62a9f03bbfd71d764baccd44699cdcdeedb8db1dd4429e0646b2/PyQt6_sip-13.0.1.tar.gz"
+    sha256 "310d0c0efdbc086ef913f4ecdfaf8e62f2b70cadeacb42536bfe943c19709987"
   end
 
   def install
-    version = Language::Python.major_minor_version Formula["python@3.9"].opt_bin/"python3"
-    args = ["--confirm-license",
-            "--bindir=#{bin}",
-            "--destdir=#{lib}/python#{version}/site-packages",
-            "--stubsdir=#{lib}/python#{version}/site-packages/PyQt5",
-            "--sipdir=#{share}/sip/Qt5",
-            # sip.h could not be found automatically
-            "--sip-incdir=#{Formula["sip"].opt_include}",
-            "--qmake=#{Formula["qt@5"].bin}/qmake",
-            # Force deployment target to avoid libc++ issues
-            "QMAKE_MACOSX_DEPLOYMENT_TARGET=#{MacOS.version}",
-            "--designer-plugindir=#{pkgshare}/plugins",
-            "--qml-plugindir=#{pkgshare}/plugins",
-            "--pyuic5-interpreter=#{Formula["python@3.9"].opt_bin}/python3",
-            "--verbose"]
-
-    system Formula["python@3.9"].opt_bin/"python3", "configure.py", *args
-    system "make"
-    ENV.deparallelize { system "make", "install" }
+    args = %W[
+      --target-dir #{prefix}
+      --no-make
+      --confirm-license
+    ]
+    system "sip-build", *args
+    cd "build" do
+      system "make"
+      qt_cellar = "#{HOMEBREW_CELLAR}/qt/#{Formula["qt"].version}"
```

Comment:
Is this not the same thing as `Formula["qt"].prefix`?

Reply:
Thanks, add a version parameter.